core/state/statedb.go

```diff
@@ -877,3 +889,30 @@ func (s *StateDB) Commit(deleteEmptyObjects bool) (common.Hash, error) {
 	}
 	return root, err
 }
+
+func (s *StateDB) AddAccessListAccount(addr common.Address) {
+	if s.accessList.AddAddr(addr) {
+		s.journal.append(accessListAddAccountChange{&addr})
+	}
+}
+
+func (s *StateDB) AddAccessListSlot(addr common.Address, slot common.Hash) {
+	addrCh, slotCh := s.accessList.AddSlot(addr, slot)
+	if addrCh {
+		s.journal.append(accessListAddAccountChange{&addr})
+	}
```

Comment:
Question: can it happen that we have an untouched address when we touch a slot?

Reply:
I guess it's better to be safe than sorry, but might be good to spec exactly when it can happen - if it can.

Reply:
No. If a slot-touch happens first, then we internally make two events out of it

Reply:
I'm not sure I follow. This method here specified exactly what happens when you touch a slot. It explicitly adds the address if it's not already there. What do you mean??

Reply:
Now I get your question. Answer is "No" - as far as I know, this cannot happen. ( ut when fuzzing journal-events, we can synthetically trigger it)